### ISSUE TITLE
Character style 'strong' for TextRun

### ISSUE THREAD

**Issue by deathg0d** [dolanmiu/docx#727]
![image](https://user-images.githubusercontent.com/2324523/102906876-59147800-449d-11eb-826a-3df6e5854992.png)

I have a paragraph block which has two `TextRun`s in it, for example:
```
new Paragraph ({
    children: [
        new TextRun({
            text: "First Word",
            bold: true
        }),
        new TextRun(" - This text was created using heading 3 paragraph style. It has a slight left indentation for lines except the first one.")
    ]
})
```

Currently I am using the bold parameter but I want to use the "strong" character style. This is because styles can be manipulated from `Design > Quick Parts > Field`. Any way to do this?

**Comment by dolanmiu** [comment]
I have updated the following demo for your use case:

https://github.com/dolanmiu/docx/blob/master/demo/51-character-styles.ts#L40-L52

You would need to define your own character styles as this is a `character style`, which `docx` currently doesnt have defaults for. `docx` currently only have Paragraph style defaults.

Having said this, `strong` isn't a default paragraph style either lol, fixed in the following PR:

https://github.com/dolanmiu/docx/pull/728

**Comment by deathg0d** [comment]
Thanks for the quick PR! When do you think these changes will be available through npm?

**Comment by dolanmiu** [comment]
in `5.5.0` in the next release

maybe in a few weeks time

**Comment by deathg0d** [comment]
@dolanmiu Hi, I have tried this update and I still am not able to get it to work. 
Here's what I want to achieve: [Auto-header-test.docx](https://github.com/dolanmiu/docx/files/6151301/Auto-header-test.docx). Basically in the header section, on the left side I want to automatically select the first word with strong type and on the right side I want to select the last word with strong type on the page. 

You would generally use `Design > Quick Parts > Field >> StyleRef > strong` to do this. However, I haven't been able to achieve this through this module yet.

**Comment by dolanmiu** [comment]
Oh theres an easy way to do that:

```ts
new Paragraph({
    children: [
        new TextRun({
            text: "Word1: ",
            bold: true,
        }),
        new TextRun("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras pretium urna id egestas volutpat. Nam ut mi sit amet leo fermentum blandit. Nullam non eros a odio accumsan rhoncus. Praesent dapibus laoreet congue. Nulla non libero blandit, gravida ligula eget, ullamcorper nisi. Aenean euismod odio ex, in auctor dolor rutrum sit amet. Vivamus tempus quam at magna tincidunt blandit sit amet non tellus. Sed pulvinar mi at velit luctus molestie. Cras feugiat mauris eget bibendum dapibus. In dapibus maximus mauris quis dapibus. Duis facilisis, neque at elementum sodales, turpis sapien ultricies turpis, vel egestas lectus ligula sed lectus."),
    ],
}),
```

Essentially, make multiple TextRuns within a Paragraph

**Comment by deathg0d** [comment]
@dolanmiu That's not what I meant. That I know how to do. Automatically setting the headers is where I am stuck:
![image](https://user-images.githubusercontent.com/2324523/111447996-1aa5f680-8736-11eb-8a95-efc12cb3ee68.png)

**Comment by dolanmiu** [comment]
You may have to do that programatically, get the last item on your array, then set it in the header

then use `tab stops` to position the text to the right etc

**Comment by deathg0d** [comment]
That's difficult to implement because last element of the array may not be the last element of the page. On every page, headers should reflect word heading of that **particular page only**.  If you download and check the doc file I previously posted you'll notice that the headers are dynamic. They are different for every page and only correspond to words in that particular page.

Word provides this feature. To place dynamic components like these in word, I can simply go to `Design > Quick Parts > Field >> StyleRef > strong`. This will automatically select the first `strong` styled phrase of the page (there's another option to select last `strong` styled phrase). However, `StyleRef` requires paragraph or header **styles** but `bold` is not a style and hence is not selectable. We need a `strong` style, which is the reason for this thread.

In the word file I posted, you'll find that the paragraph style is `Normal` but the first word has character style `strong`. This is why I am able to create the dynamic headers.

With docx plugin, I am not able to set character style to strong. I created a docx file using version 5.5.0 and the following code for the first word from [here](https://github.com/dolanmiu/docx/blob/master/demo/51-character-styles.ts#L40-L52):

```
new TextRun({
    text: 'word1',
    style: "strong"
})
```
Once the doc is created, If i select the word and check the style, it still shows `Normal` not `Strong`. If I can set character style to `Strong` that would solve first part of the problem for now at least.
